### ISSUE TITLE
Mailing Address Info in Assignment Notices

### ISSUE THREAD

**Issue by jlbooker** [AppStateESS/homestead#953]
Once the package desk issue is complete (#947), add package desk address (depending on the student's assignment) and PO Box # to the assignment email. Use the company logos and some good design to make it painfully obvious which address is for which type of carrier (PO Box is '''only''' for things shipped with the postal service, package desk is '''only''' for UPS/FedEx).

**Comment by jlbooker** [comment]
Duplicate of #595
